vision/mnist/DCGAN/dcgan.md

```diff
@@ -0,0 +1,195 @@
+# ***Generative Adversarial Network Tutorial***
```

Comment:
Can we just use normal headings for these rather than the extra formatting / html tags?